inf-clojure.el

```diff
@@ -726,7 +726,10 @@ If you are using REPL types, it will pickup the most approapriate
 (define-obsolete-variable-alias 'inf-clojure-completion-command 'inf-clojure-completion-form "2.0.0")
 
 (defcustom inf-clojure-completion-form-lumo
-  "(doall (map str (lumo.repl/get-completions \"%s\")))"
+  "(let [ret (atom)]
```

Comment:
If you set it to:

`(do (set! lumo.repl/*pprint-results* false) (let [hack (atom)] (lumo.repl/get-completions "redu" #(reset! hack (map str %))) @hack))`

and squash all commits into one, we are good :smile: